site/content/en/docs/Advanced/allocator-service.md

```diff
@@ -53,9 +53,7 @@ If the `agones-allocator` service is installed as a `LoadBalancer` [using a rese
 
 ```bash
 EXTERNAL_IP=$(kubectl get services agones-allocator -n agones-system -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-helm upgrade --install --wait \
-   --set agones.allocator.service.loadBalancerIP=${EXTERNAL_IP} \
-   ...
+helm upgrade my-release agones/agones -n agones-system --wait --set agones.allocator.service.loadBalancerIP=${EXTERNAL_IP}
```

Comment:
I think that the `...` here was to indicate that if you had passed any custom variables during your initial helm install you need to pass them again during the upgrade (so as not to revert to the default values). Your change makes this command copy/pastable if you do the default helm install, which is nice, but it would be good to convey the `...` (in a way that is clearer that the elipsis did before).